lib/posix/posix.nim

```diff
@@ -663,6 +663,25 @@ proc waitid*(a1: cint, a2: Id, a3: var SigInfo, a4: cint): cint {.
 proc waitpid*(a1: Pid, a2: var cint, a3: cint): Pid {.
   importc, header: "<sys/wait.h>".}
 
+type Rusage* {.importc: "struct rusage", header: "<sys/resource.h>",
+               pure, final.} = object
```

Comment:
Instead of ``pure, final`` use ``bycopy`` please.

Reply:
done.

Reply:
Anything else?

Reply:
Could have a ``changelog.md`` entry.